README.md

```diff
@@ -1,32 +1,32 @@
-# Linuxbrew-fonts
+# Fonts for Homebrew on Linux
 
-Linuxbrew-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-fonts/).
-
-Linuxbrew-fonts is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux.
+homebrew-linux-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts). 
+It is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux since Homebrew Casks are only for macOS.
 
 ## Prerequisites
 
-Fonts are available through homebrew and linuxbrew. So, first you must install [Homebrew](http://brew.sh) or [Linuxbrew](http://linuxbrew.sh). However this repository is **NOT** an "external command” of Homebrew, it is a different from the upstream repository, homebrew/homebrew-cask-fonts. Please caught of that.
-
-**Since Homebrew 1.9 or later supports the both platform, you can use this repository with homebrew in Linux too**
+These fonts are installable with [Homebrew](http://brew.sh).
+This repository is **NOT** an "external command” of Homebrew.
+It is different from the upstream repository, `homebrew/homebrew-cask-fonts`.
+Please be aware of that.
 
 ## Let’s try it!
 
 ```bash
-$ brew tap linuxbrew/fonts                 # you only have to do this once!
+$ brew tap homebrew/linux-fonts          # you only have to do this once!
 $ brew install font-inconsolata --HEAD
 ```
 
 ## Working with fontconfig
 
-If you're using major distributions, in single-user installation just type;
+If you're using major distributions in a single-user installation, run:
 
 ```
 $ ln -s /home/linuxbrew/.linuxbrew/share/fonts ~/.local/share/fonts
 $ fc-cache -fv
 ```
 
-For multi-user installation (requires sudo) type;
+For multi-user installation, you must use `sudo` to run:
```

Comment:
This should say either "For multi-user installation**s**" or "For **a** multi-user installation".

Reply:
Also, I believe this ending would read better:

```suggestion
For multi-user installation, you must use `sudo` to run the commands:
```

Reply:
Hey @colindean, please note that applying the suggestion marked this thread as resolved, but the first comment still applies.